src/plugins/wordBreak.js

```diff
@@ -1,12 +1,14 @@
 export default function() {
-  return function({ addUtilities, variants }) {
+  return function({ addUtilities, variants, target }) {
+    const wrapPropertyName = target('wordBreak') === 'ie11' ? 'word-wrap' : 'overflow-wrap'
```

Comment:
I left this using `overflow-wrap` when not targeting ie11 since that's the official name of the property now. Technically it would be fine to just use `word-wrap` instead of `overflow-wrap` if I'm reading https://css-tricks.com/almanac/properties/o/overflow-wrap/#the-historical-word-wrap-property right. Do ya'll think I made the right call here?